test-cases/longevity/longevity-cdc-basic-24h.yaml

```diff
@@ -0,0 +1,63 @@
+test_duration: 420
+
+latte_schema_parameters:
+  lwt: 'false'
+  cdc: 'true'
+  mv: 'false'
+  ks_name: 'cdc_keyspace'
+  table_name: 'cdc_table'
+
+prepare_write_cmd:
+  - >-
+    latte run -f write_to_table -P rows_per_partition=10 -P rows=5000000
+    --duration=250000 --concurrency=1000 --rate=2500 --start-cycle 0 --tag write-1
+    data_dir/latte/tablet_split_merge_load.rn
+
+  - >-
+    latte run -f write_to_table -P rows_per_partition=10 -P offset=5000000 -P rows=5000000
+    --duration=250000 --concurrency=1000 --rate=2500 --start-cycle 500000001 --tag write-2
+    data_dir/latte/tablet_split_merge_load.rn
+
+stress_cmd:
+  - >-
+    latte run -f "update_or_delete_by_cycle" --consistency=QUORUM --concurrency 100 --threads=50 --rate=25000 --duration=360m --tag update-or-delete-by-cycle
+    --request-timeout=900s --retry-number=50 --retry-interval=1s,30s
+    -P offset=10000000 -P rows=50000000 -P rows_per_partition=1
+    -P update_cycles=1 -P read_cycles=2 -P delete_cycles=1
+    data_dir/latte/tablet_split_merge_load.rn
+
+  - >-
+    latte run -f "update_or_delete_by_cycle" --consistency=QUORUM --concurrency 100 --threads=50 --rate=25000 --duration=360m --tag update-or-delete-by-cycle
+    --request-timeout=900s --retry-number=50 --retry-interval=1s,30s
+    -P offset=100000000 -P rows=50000000 -P rows_per_partition=1
+    -P update_cycles=1 -P read_cycles=2 -P delete_cycles=1
+    data_dir/latte/tablet_split_merge_load.rn
+
+
+db_type: mixed_scylla
```

Comment:
what is mixed scylla?

Reply:
we have oracle_db_nodes here

Reply:
For the CDC test, SCT configures a secondary cluster using Oracle. This cluster serves for data replication. The CDC replicator reads data from the *_scylla_cdc_log_ table and transfers it to the Oracle cluster. Subsequently, SCT compares the data in the test cluster and the Oracle cluster to ensure they are identical. This verification confirms that the Scylla CDC feature operates as intended.